pydocx/DocxParser.py

```diff
@@ -39,7 +39,6 @@
 )
 from pydocx.wordml import WordprocessingDocument
 
-logging.basicConfig(level=logging.DEBUG)
```

Comment:
This should be moved to being called inside [main](https://github.com/CenterForOpenScience/pydocx/blob/master/pydocx/__init__.py#L22). We want to config the logger for CLI usage, still.